modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -3,88 +3,609 @@
+| Option           | `font_formats`       | `font_family_formats`
+
+| Toolbar item     | `fontselect`         | `fontfamily`
+
+| Menu item        | `fontformats`        | `fontfamily`
+
+| Option           | `fontsize_formats`   | `font_size_formats`
+
+| Toolbar item     | `fontsizeselect`     | `fontsize`
+
+| Menu item        | `fontsizes`          | `fontsize`
+
+| Toolbar item     | `formatselect`       | `blocks`
+
+| Menu item        | `blockformats`       | `blocks`
+
+| Toolbar item     | `styleselect`        | `styles`
+
+| Menu item        | `formats`            | `styles`
+
+| Option           | `lineheight_formats` | `line_height_formats`
+|===
+
+NOTE: The behaviour of these elements is unchanged.
+
+=== `primary`
+
+The `primary` property was a boolean property available as a basic panel component of `dialogs`.
+
+It has been deprecated and replaced by the new `buttonType` property.
+
+
+[[ui-elements-and-components-tables-positioned-using-margins-not-float]]
+=== Tables are now positioned using `margin`, not `float`
+
+{productname} previously used `float` to position tables to the left or right of the viewport (the working area available to the {productname} editor). This caused multiple layout and UX problems.
+
+For example, consider a {productname} document that included a table that took up less than 100% of the viewport and a paragraph of text below that table. If both the table and the paragraph were selected and their *alignment* was changed to _left_ or _right_, the paragraph of text moved up beside the table.
+
+As another example, consider a {productname} document containing a table that is wider than the viewport. If you selected this table’s contents and set the *alignment* to _right_, the portion of the table that was outside the viewport disappeared to the left, and no scrollbar presented allowing you to bring that portion back into view.
+
+As of {productname} 6.0, tables are positioned using `margin` rather than `float`. This prevents these layout and UX problems occurring.
+
+However, when upgrading to {productname} 6.0, there are two things to be aware of:
+
+First, {productname} 6.0 does not alter existing documents. Extant {productname} documents with tables that were aligned to either _left_ or _right_ will still have the `float` property assigned when opened and edited unless and until their *alignment* is altered.
+
+If an extant {productname} document with an aligned table is opened using {productname} 6.0 and the *alignment* of the table is re-set, {productname} 6.0 will remove the `float` property and use the `margin` property to set the *alignment*.
+
+Second, if the previous behaviour is required, a custom `format` will need to be created. Specifically, the following CSS (the https://github.com/tinymce/tinymce/blob/release/5.10/modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts#L27-L117[table-specific CSS] from {productname} 5.x’s `DefaultFormats.ts`) must be added to the `format` configuration:
```

Comment:
Formats and CSS are two different things :)
```suggestion
Second, if the previous behaviour is required, a custom `format` will need to be created. Specifically, the following formats (the https://github.com/tinymce/tinymce/blob/release/5.10/modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts#L27-L117[table-specific code] from {productname} 5.x’s `DefaultFormats.ts`) must be added to the `format` configuration:
```

Reply:
Just to clarify:

The appended example code is straight `css`.

*But*, in the context of said `css` being added to a custom `format` configuration, said example should be called `code` and said example should be thought of as a collection of custom _formats_.

Am I reading this right?

Reply:
But that's my point - the embedded example is _not_ css, it's a JS format definition.